### DISCUSSION TITLE
Storylet DDR: There is an issue, the gate won't open

### DISCUSSION THREAD

**Discussion by mairin** [RedHatOfficial/creative#18] [Ideas]
Storylet Design Decision Record: There is an issue, the gate won't open
=============================

Related
-------
See #17 

Synopsis
------
This is the expansion of the following storylet idea:

> Each of the color wizards are stationed at one of the three color gates alongside their entourages. Using magic mirrors to communicate, 
> they count down 3-2-1 to cast the spell simultaneously at each of the 3 gates. You're with the red wizard. The wizards cast the spell in 
> unison - but the gate doesn't open.

Puzzle Dependency Graph
-----
![Screenshot from 2022-09-14 17-54-31](https://user-images.githubusercontent.com/799683/190269923-6375a376-e4e1-4920-a157-eb7f4b9b8b5f.png)

* [Leave inline comments on graph here](https://design.penpot.app/#/view/63fb2ba0-3470-11ed-9e6a-c67f7655c7b3?page-id=63fb2ba1-3470-11ed-9e6a-c67f7655c7b3&section=interactions&index=0&share-id=1ad41560-3478-11ed-9e6a-c67f7655c7b3)
* [Editable graph source in Penpot](https://design.penpot.app/#/workspace/054c9580-3470-11ed-9e6a-c67f7655c7b3/63fb2ba0-3470-11ed-9e6a-c67f7655c7b3?page-id=63fb2ba1-3470-11ed-9e6a-c67f7655c7b3&layout=layers)

**Comment by dvanduzer** [comment]
As a player, we are focused on opening just one of the three gates into Citadel City. Somehow I want to work a joke about the One Token Ring.
